lib/getUrisFromReq.js

```diff
@@ -23,6 +25,7 @@ function getUrisFromReq (req, res) {
       designId = designId + '/' + req.params.version
     }
     url = '/user/' + encodeURIComponent(req.params.userId) + '/' + designId
+    baseUrl = '/user/' + encodeURIComponent(req.params.userId) + '/' + req.params.collectionId
```

Comment:
These strings are a bit confusing, they could be rewritten to have subparts clearly shown in larger parts:
```
let baseUrl = '/user/' + encodeURIComponent(req.params.userId) + '/' + collectionId
let url = baseUrl + '/' + req.params.displayId /* maybe + version */
let baseUri = config.get('databasePrefix') + baseUrl
```

etc. etc. Not really a requirement but could make the relationship between these parts easier to understand.